WordPress/wordcamp.org
# Add infrastructure to run PHPUnit

This adds a job to run PHPUnit on travis, and adds an optional docker file to set up tests & PHPUnit locally. Previously it was not clear how to run the few tests we have, so this also updates [the documentation.](https://github.com/WordPress/wordcamp.org/tree/add/travis-phpunit/.docker#testing-with-phpunit) Finally, the Remote CSS test was failing after the change in #222, so the test has been fixed.

Having this testing infrastructure will make it easier to write & run tests in the future. New tests will need to be added to the `phpunit.xml.dist` file, following the other `<testsuite>` definitions.

Some past effort here: #105

### How to test the changes in this Pull Request:

1. Verify that the tests ran & passed on travis by looking at the Checks tab ^
2. Try installing this locally by following [the instructions](https://github.com/WordPress/wordcamp.org/tree/add/travis-phpunit/.docker#testing-with-phpunit)
3. If you already had `phpunit` running some other way, make sure it still works
